.travis.yml

```diff
@@ -42,12 +39,11 @@ install:
   - nvm use $TRAVIS_NODE_VERSION
 
   # Install requirements
-  - pip install -r requirements_devel.txt
+  - pip install -r requirements.txt
```

Comment:
any reason why this was changed from `requirements_devel.txt`? It's importing `requirements.txt` so just ensure no packages are repeated in both.